specification/semantic_conventions.md

```diff
@@ -18,6 +18,7 @@ All Splunk distributions of OpenTelemetry,
 - SHOULD set the following resource attributes when applicable:
   - `telemetry.auto.version`
 
+Note: this section does not apply to Real User Monitoring libraries, as they do not use the OpenTelemetry Resource.
```

Comment:
I think that we should be setting the `service.name` on the Resource, since that part does get mapped into the zipkin spans, and is visible in the RUM UI currently.

Reply:
We don't do it anywhere right now, do we?
Maybe we should replace the `applicationName` config with `serviceName` and use it as `service.name`, not `app`

Reply:
https://github.com/signalfx/splunk-otel-android/blob/main/splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java#L132

Reply:
iOS uses `app`: https://github.com/signalfx/splunk-otel-ios/blob/main/SplunkRumWorkspace/SplunkRum/SplunkRum/InstrumentationUtils.swift#L71
JS-web seems to be using `app` too: https://github.com/signalfx/splunk-otel-js-web/blob/main/src/SplunkWebTracerProvider.ts#L53

Reply:
Hmm, so does android actually: https://github.com/signalfx/splunk-otel-android/blob/main/splunk-otel-android/src/main/java/com/splunk/rum/RumAttributeAppender.java#L56

If zipkin supports `service.name` then maybe we should switch to it in all RUM libs?

Reply:
I think we need both at the moment, unfortunately.

Reply:
😢 
We can make setting both `service.name` and `app` mandatory for now, as a preparation to switching to `service.name` in the future (if we ever are going to do that).

Reply:
We(web rum) have a whole epic dedicated to matching OTEL semconv. Currently I think setting [service.* attributes on Resource](https://signalfuse.atlassian.net/browse/APMI-1723) is being worked on. @t2t2 knows more.

Reply:
> 😢
> We can make setting both `service.name` and `app` mandatory for now, as a preparation to switching to `service.name` in the future (if we ever are going to do that).

let's not do this -- user should set one thing: `service.name` -- if `app` is also required we should just set that for the user based on the specified `service.name`.

Reply:
> let's not do this -- user should set one thing: `service.name` -- if `app` is also required we should just set that for the user based on the specified `service.name`.

No, what I meant by my previous comment is "RUM library should set two attributes" - user only has to set `applicationName`/`serviceName` property, its value just get placed in two attributes.